tests/flow/test_function_calls.py

```
@@ -577,55 +577,55 @@ def test18_allocated_keys(self):
 
-    #    # Test has labels using functions mismatch type
-    #    query = """MATCH (n) WHERE hasLabels(n, ['person', 1]) RETURN n.name"""
-    #    try:
-    #        self.graph.query(query)
-    #    except redis.ResponseError as e:
-    #        self.env.assertContains("Type mismatch: expected String but was Integer", str(e))
+        # Test has labels using functions mismatch type
+        query = """MATCH (n) WHERE hasLabels(n, ['person', 1]) RETURN n.name"""
+        try:
+            self.graph.query(query)
+        except redis.ResponseError as e:
```

Ruff (0.14.10)

`redis` may be undefined, or defined from star imports

(F405)